src/report/ReportManager.ts

```diff
@@ -868,8 +875,8 @@ class DisplayManager {
             body: htmlToText(document.body.outerHTML, { wordwrap: false }),
             format: "org.matrix.custom.html",
             formatted_body: document.body.outerHTML,
+            [ABUSE_REPORT_KEY]: report
```

Comment:
No brackets.

Reply:
if there's no brackets then it will be read as a bare string surely

Reply:
My bad!

I strongly dislike this JS idiom (because it's abusing array to string conversion) but I don't think that there is any better option.